### PR TITLE
[ADD] product_multi_company

### PR DESCRIPTION
**Pull request by pedrobaeza**
# 
# Product permissions for discrete companies

This modules allows to select in which of the companies you want to use each
of the products.
# Installation

This module uses the post an uninstall hooks for updating default product
template security rule. This only means that updating the module will not
restore the security rule this module changes. Only a complete removal and
reinstallation will serve.
# Usage

On the product form view, go to the "Information" tab, and put the companies
in which you want to use that product. If none is selected, the product will
be visible in all companies. The default value is the current company.

### CODE REVIEW

**Comment by rvalyi** [comment]
:+1: thanks looks useful and cleanly coded.

File: product_multi_company/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Product permissions for discrete companies
+==========================================
+
+This modules allows to select in which of the companies you want to use each
+of the products.
+
+Installation
+============
+
+This module uses the post an uninstall hooks for updating default product
```
Comment (elicoidal) [inline product_multi_company/README.rst]: s/an/and

File: product_multi_company/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+template security rule. This only means that updating the module will not
+restore the security rule this module changes. Only a complete removal and
+reinstallation will serve.
+
+Usage
+=====
+
+On the product form view, go to the "Information" tab, and put the companies
+in which you want to use that product. If none is selected, the product will
+be visible in all companies. The default value is the current company.
```
Comment (elicoidal) [inline product_multi_company/README.rst]: s/all companies/all of them
s/current company/current one

File: product_multi_company/README.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+------------
+
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
```
Comment (elicoidal) [inline product_multi_company/README.rst]: https

**Comment by elicoidal** [comment]
LGTM

**Comment by legalsylvain** [comment]
:+1: Very usefull module indeed.

**Comment by lmignon** [comment]
:+1: You save me at least one day of dev... I need it (code review + functional)

**Comment by pedrobaeza** [comment]
@elicoidal, fixed!

I think this one is ready to  be merged.

**Comment by rvalyi** [comment]
more than 3 approvals, right? -> merge 1st, regret later.

**Pull request merged**